src/arch/amdgpu/vega/insts/sopc.cc

```diff
@@ -584,8 +584,8 @@ namespace VegaISA
     void
     Inst_SOPC__S_CMP_LG_U64::execute(GPUDynInstPtr gpuDynInst)
     {
-        ConstScalarOperandU64 src0(gpuDynInst, instData.SSRC0);
-        ConstScalarOperandU64 src1(gpuDynInst, instData.SSRC1);
+        ConstScalarOperandI64 src0(gpuDynInst, instData.SSRC0);
```

Comment:
Why is this a bug?  The instruction is U64, not I64 ...

Reply:
This *should* still work? Unless the compiler sign extends. Probably safer to make a new PR just removing the real problem.